testing/dart/image_filter_test.dart

```diff
@@ -170,6 +182,24 @@ void main() {
     checkBytes(bytes, greenCenterBlurred, greenSideBlurred, greenCornerBlurred);
   });
 
+  test('ImageFilter - dilate', () async {
+    final Paint paint = Paint()
+      ..color = green
+      ..imageFilter = makeDilate(1.0, 1.0);
+
+    final Uint32List bytes = await getBytesForPaint(paint);
+    checkBytes(bytes, green.value, green.value, green.value);
```

Comment:
This makes it look like the operation is just a fill operation. Is there a way we can make this test more robust?

Reply:
I suppose this is showing the effect as it copied (dilated) the center green pixel out to the sides and corners (this seems odd that the parameters are called radii then as that would imply a circular dilation, but this seems to be doing a box dilation?)

Reply:
I guess the term radius comes from SVG which uses that name even though the effect is a box filter.

Reply:
> I suppose this is showing the effect as it copied (dilated) the center green pixel out to the sides and corners (this seems odd that the parameters are called radii then as that would imply a circular dilation, but this seems to be doing a box dilation?)

Yes I guess it is box dilation. The reason may be, box is much, much faster than circular dilation (as I have learnt in computer vision courses).